compiler/rustc_mir_dataflow/src/elaborate_drops.rs

```diff
@@ -738,8 +739,13 @@ where
         loop_block
     }
 
-    fn open_drop_for_array(&mut self, ety: Ty<'tcx>, opt_size: Option<u64>) -> BasicBlock {
-        debug!("open_drop_for_array({:?}, {:?})", ety, opt_size);
+    fn open_drop_for_array(
+        &mut self,
+        array_ty: Ty<'tcx>,
+        ety: Ty<'tcx>,
```

Comment:
If you're accepting the type of the array would it be reasonable to just compute `ety` internally instead of accepting a redundant parameter. Though it doesn't seem like this has many callers so I doubt this would cause any problems lol